spec/models/organization_spec.rb

```diff
@@ -28,6 +28,43 @@
 
 RSpec.describe Organization, type: :model do
   let(:organization) { create(:organization) }
+
+  describe "validations" do
+    it do
+      is_expected.to validate_numericality_of(:deadline_day)
+        .only_integer
+        .is_greater_than_or_equal_to(1)
+        .is_less_than_or_equal_to(28)
+        .allow_nil
+    end
+
+    it do
+      is_expected.to validate_numericality_of(:reminder_day)
+        .only_integer
+        .is_greater_than_or_equal_to(1)
+        .is_less_than_or_equal_to(28)
+        .allow_nil
+    end
```

Comment:
Honestly I find the previous way of doing things a bit more explicit and obvious - putting in the specific values that are or aren't allowed. This seems like it's just testing that Rails validations work as specified. Others might disagree with that. 🤷

Reply:
I could go either way; I agree that it's sometimes hard to see the benefit in shoulda matchers, but they do provide a concise way to ensure no regressions occur.  I can change it if you like, though.

Reply:
Note: until answered, I ported these as is into the new Deadlinable spec.